src/org/safs/rest/service/Headers.java

```diff
@@ -409,24 +412,26 @@ public static String convertHeadersMapToMultiLineString(Map<String,String> heade
 	private static Header[] parseHeadersInMultiLineString(String _headerString) {
 		
 		/*
-		 * This is pretty close to how Apache HTTP Client5 parses headers.
+		 * This is pretty close to how Apache HTTP Core5 parses headers.
 		 */
 		try {
 			ByteArrayInputStream inputStream = new ByteArrayInputStream(_headerString.getBytes("UTF-8"));
 			
 			int buffersize = org.apache.hc.core5.http.config.ConnectionConfig.DEFAULT.getBufferSize();
-			HttpTransportMetricsImpl inTransportMetrics = new HttpTransportMetricsImpl();
-			MessageConstraints messageConstraints = MessageConstraints.DEFAULT;
-			SessionInputBufferImpl inbuffer = new SessionInputBufferImpl(inTransportMetrics, buffersize, -1, messageConstraints, null);
-			BasicLineParser parser = org.apache.hc.core5.http.message.BasicLineParser.INSTANCE;
+			BasicHttpTransportMetrics inTransportMetrics = new BasicHttpTransportMetrics();
+			final H1Config h1Config = H1Config.DEFAULT;
+			final CharsetDecoder chardecoder = null;
+			SessionInputBufferImpl inbuffer = new SessionInputBufferImpl(inTransportMetrics, buffersize, -1, 
+			                                                             h1Config.getMaxLineLength(), chardecoder);
+			final LineParser lineParser = BasicLineParser.INSTANCE;
 			
-			inbuffer.bind(inputStream);
-			List<CharArrayBuffer> headerLines = new ArrayList<CharArrayBuffer>();
-			Header[] headers = org.apache.hc.core5.http.impl.io.AbstractMessageParser.parseHeaders(
+			List<CharArrayBuffer> headerLines = new ArrayList<>();
+			final Header[] headers = AbstractMessageParser.parseHeaders(
 				inbuffer,
-				messageConstraints.getMaxHeaderCount(),
-				messageConstraints.getMaxLineLength(),
-				parser,
+				inputStream,
+				h1Config.getMaxHeaderCount(),
+				h1Config.getMaxLineLength(),
+				lineParser,
 				headerLines
```

Comment:
As you can see, there were a lot of changes between alpha1 and alpha2 in the way headers are parsed.